.babelrc

```diff
@@ -0,0 +1,12 @@
+{
+    "presets": [
+        [
+            "@babel/preset-env",
```

Comment:
As noted by @salzhrani in https://github.com/dwyl/hapi-postgres-connection/pull/42#issuecomment-617642140 we probably don't need Babel given that Hapi 19 requires Node.js 12+.

Reply:
I don't know why, but when i try do travis tests i see it
![image](https://user-images.githubusercontent.com/20814779/79970577-c8b00e80-849b-11ea-8f99-96125912edf0.png)

like travis run tests using old node version - but i try check node version - it is 12 

I don't know how fix it

Reply:
@nelsonic @salzhrani I whrote the question topic https://travis-ci.community/t/node-version-12-but-used-another/8277

Reply:
I think it might have to do with Istanbul/Tap

Reply:
![image](https://user-images.githubusercontent.com/20814779/79976119-cb633180-84a4-11ea-94a9-8199b2c83f95.png)

Reply:
It seems Istanbul doesn't support [async](https://github.com/gotwarlost/istanbul/issues/733) keyword, we can either upgrade Istanbul to nyc or rewrite the function to use promises.

Reply:
@salzhrani we definitely need to upgrade from Istanbul to NYC. 
The upgrade has been smooth in our other modules/apps where we've had to do the same. 👍

Reply:
@Hydrock thanks for clarifying. 👍

Reply:
I may do this

Reply:
@nelsonic @salzhrani I'm done
https://travis-ci.org/github/dwyl/hapi-postgres-connection/builds/678340079
Istanbul and babel is removed - change to nyc 
returned index.js to root folder

Reply:
esprima is parsing tool - ofcause! https://esprima.org/
Thanks @salzhrani